Qt.py

```diff
@@ -60,6 +61,12 @@ def _pyqt4():
     PyQt4.__qt_version__ = PyQt4.QtCore.PYQT_VERSION_STR
     PyQt4.load_ui = install_load_ui(module=uic)
 
+    try:
+        sip.setapi('QString', 2)
+        sip.setapi('QVariant', 2)
+    except ValueError as error:
+        sys.stdout.write('Qt Warning: ' + str(error) + '\n')
```

Comment:
```python
sys.stdout.write('Qt Warning: ' + str(error) + '\n')`
```

This could be something like..

```python
sys.stdout.write('Qt Warning: Using sip API v1')
```